Sources/StreamChat/Repositories/SyncOperations.swift

```diff
@@ -6,10 +6,15 @@ import Foundation
 
 /// A final class that holds the context for the ongoing operations during the sync process
 final class SyncContext {
+    let lastSyncAt: Date
```

Comment:
I would add a more descriptive name or documentation to it. Remember how we both were understanding different things around this topic :joy: